### PR TITLE
[KOA-4703] Slider label formatter

### PR DESCRIPTION
**Pull request by seraco**
Adds documentation for explaining how to use label formatting in `Slider` component. Also updates the related stories.

allall_dm

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [ ] Tests (no changes in component code, only in stories)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ See [`CHANGELOG.md`](CHANGELOG.md) for real-world examples of good changelog ent
 
 - `bpk-component-infinity-gauntlet`:
   - New `timeStone` prop for controlling time. See &lt;link to docs site&gt;.
+- `bpk-component-slider`:
```
Comment (bvitaliyg) [inline UNRELEASED.md:31]: We usually add the entries to another place in this file – please check out the history for samples :)

**Review by bvitaliyg — CHANGES_REQUESTED** [review]
Thank you for your contribution! 🎉 🎒

Just a small review comment. Also, we use a script to take docs screenshots automatically, so the badges screenshot will be overridden in future – can we make it without the interaction?

File: UNRELEASED.md (hunk 1 above)
Comment (seraco) [reply UNRELEASED.md]: Addressed by 70f79058d0e1eb33ed92033bb0630c6939103cc2.

**Review by bvitaliyg — APPROVED** [review]
👍

**Pull request merged**